src/core/runtime.ts

```diff
@@ -160,6 +161,20 @@ class Runtime {
         }
     }
 
+    private handlePageScrollPosition(): void {
+        if (window.location.hash) {
+            /** Scroll to hash element */
+            const element = document.body.querySelector(window.location.hash);
+            if (element) {
+                const elementYPosition = (element.getBoundingClientRect().top - document.body.getBoundingClientRect().top) as number;
```

Comment:
Was there any particular reason we're calculating the offset on the element instead using the `scrollIntoView()` method?

Ref: [Element API](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView).